platform/event/src/main/java/org/motechproject/event/listener/annotations/types/MotechListenerType.java

```diff
@@ -1,4 +1,4 @@
-package org.motechproject.event.listener.annotations;
+package org.motechproject.event.listener.annotations.types;
```

Comment:
what's the point of this new package? It just contains 1 class, and both the old one and the new one are still exported.